lucien-ios/Models/User.swift

```diff
@@ -30,3 +30,14 @@ struct User: Decodable {
         case updatedAt = "updated_at"
     }
 }
+
+struct User2: Decodable {
```

Comment:
It's not good practice to name a struct the same as another with a number attached. name it more descriptively

Reply:
That struct is never used anywhere else. It was used for testing purposes. I'll remove it. Thanks for catching that!